### PR TITLE
Duplicate operator fix

### PR DESCRIPTION
**Pull request by edcdavid**

### CODE REVIEW

File: deploy-operator.sh [hunk 1]
```diff
@@ -78,8 +78,11 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
+    test-network-function.com/subscription_name: '["nginx-operator-v0-0-1-sub"]'
   name: nginx-operator.v0.0.0
   namespace: placeholder
+  labels:
+    operators.coreos.com/nginx-operator.tnf: ""
```
Comment (jun-rh) [inline deploy-operator.sh:85]: this replaces test-network-function.com/operator=target?

Comment (edcdavid) [reply deploy-operator.sh:85]: yes

Comment (edcdavid) [reply deploy-operator.sh:85]: need to add the variable for namespace, will fix

Comment (jun-rh) [reply deploy-operator.sh:85]: how does autodiscovery work with a non-generic label like this?

Comment (edcdavid) [reply deploy-operator.sh:85]: need to recheck this

**Review by wying3 — APPROVED** [review]

File: deploy-operator.sh (hunk 1 above)
Comment (edcdavid) [reply deploy-operator.sh]: done

**Review by jun-rh — APPROVED** [review]

**Pull request merged**